kratos/includes/model_part.h

```diff
@@ -2012,23 +2014,23 @@ class KRATOS_API(KRATOS_CORE) ModelPart final
         }
     };
 
-    template<class TContainerType, class TIterator>
+    template<class TContainerType>
     static bool IsSubSet(
         const TContainerType& rContainer,
-        TIterator begin,
-        TIterator end)
+        typename TContainerType::iterator begin,
+        typename TContainerType::iterator end)
```

Comment:
have you compiled this locally? I imagine argument deduction would fail because the iterators are nested types of the template parameter.

Reply:
yep. It works. :)